src/lib/zkapps_examples/zkapps_examples.ml

```diff
@@ -569,19 +586,23 @@ let compile :
       | [] ->
           []
       | prover :: provers ->
+          (* here's the wrapping of prover *)
           let prover ?handler () =
             let open Async_kernel in
+            (* run the prover *)
             let%map ( _stmt
                     , ( { account_update; account_update_digest; calls }
                       , auxiliary_value )
                     , proof ) =
               prover ?handler ()
             in
+            (* create account update *)
```

Comment:
What about `let account_update = ... in` wasn't clear enough without this comment?